src/plugins/ruby/README.md

```diff
@@ -0,0 +1,141 @@
+- infos = Information about RUBY plugin is in keys below
+- infos/author = Bernhard Denner <bernhard.denner@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage
+- infos/placements = getstorage setstorage
+- infos/recommends =
+- infos/status = configurable maintained memleak
```

Comment:
Since this plugin is still in it’s early development I would recommend we add the tag `experimental` here.

Reply:
With experimental it would not be included by default.

@BernhardDenner might have used it already enough so that non-experimental is justified.

@sanssecours did you find any run-time issues except the warning?